src/index.ts

```diff
@@ -1,25 +1,40 @@
-import { readFileSync } from 'fs';
+import _chainsDeploymentInfo from '@api3dao/operations/data/chains.json';
+import { goSync, isGoSuccess } from './utils';
 
-function getServiceData(serviceName: string, chain: string) {
-  const deploymentJson = JSON.parse(readFileSync(`./deployments/${chain}.json`).toString());
-  if (!deploymentJson) {
+interface ChainInfo {
+  AirnodeRrp: string;
+  AccessControlRegistry: string;
+  OwnableCallForwarder: string;
+  RequesterAuthorizerWithManager: string;
+  RrpBeaconServer: string;
+}
+
+// Typing this more loosly to avoid TS errors
+const chainsDeploymentInfo: { [chainName: string]: ChainInfo } = _chainsDeploymentInfo;
+
+function getServiceData(apiName: string, beaconName: string, chain: string) {
```

Comment:
This means that the caller of this function will need to pass a `beaconName` like 0xce4ad44fb1f9784d6563e84d396d4e3f7d4e8fe13e02044a3f54623a03f2f8fa-get _market_spot_vwap_pairs_{pair}_latest.json, right?

Reply:
Yes, but that is to be changed in @aquarat branch.